apps/console/package.json

```diff
@@ -24,7 +24,7 @@
         "start": "run-script-os",
         "start:win32": "..\\..\\node_modules\\.bin\\webpack-dev-server --env.NODE_ENV=development --inline --hot --open",
         "start:default": "../../node_modules/.bin/webpack-dev-server --env.NODE_ENV=development --inline --hot --open",
-        "test": "jest --passWithNoTests"
+        "test": "../../node_modules/.bin/jest --passWithNoTests"
```

Comment:
Shall we add `jest` as a dev dev dependency and use `run-script-os` to make sure the command works cross platform.

```
"test": "npm run jest -- --passWithNoTests"
"jest": "run-script-os"
"jest:win32": ".\\node_modules\\.bin\\jest"
"jest:default": "./node_modules/.bin/jest"
```

Reply:
Fixed with commit c12c149